src/Sulu/Bundle/TagBundle/Tag/TagManager.php

```diff
@@ -53,16 +58,23 @@ class TagManager implements TagManagerInterface
      */
     private $domainEventCollector;
 
+    /**
+     * @var TrashManagerInterface
```

Comment:
Should be nullable the whole `TrashBundle` should be able to be removed from the `config/bundles` and all bundles should work without it. 

```suggestion
     * @var TrashManagerInterface|null
```

Reply:
why not just setting the `sulu_trash.storage.repository` to `null` instead of `doctrine`, like we did in the `ActivityBundle`?